### PR TITLE
Create node upgrade instructions and expand on key management instructions

### LINKED ISSUE #794: Doc/scripts for rotating keys (zero-downtime upgrades)
### Expected Behavior

Please describe the behavior you are expecting

### Current Behavior

What is the current behavior?

### Steps to Reproduce Behavior

How can we reproduce this?

### Logs

Are there any logs?

### System information

Run `geth version`

### PR DESCRIPTION
**Pull request by nategraf**
### Description

Expands upon existing instructions for validator signing key rotation, and adds a short page on node upgrades.

### Tested

Instructions are an expansion on a known good process, but could definitely use some 

### Related issues

- Fixes https://github.com/celo-org/celo-blockchain/issues/794

### CODE REVIEW

File: packages/docs/getting-started/glossary.md [hunk 1]
```diff
@@ -141,6 +141,10 @@ See [Block](#block).
 
 Hardware Security Module. A hardware device that hosts one or more private keys and signs data without passing the key off the device.
 
+## Key Rotation
```
Comment (nambrot) [inline packages/docs/getting-started/glossary.md:144]: Maybe link to the page?

File: packages/docs/node-operations/key-management.md [hunk 2]
```diff
@@ -1,83 +0,0 @@
...
-
-Please see the [Running a Validator](https://docs.celo.org/getting-started/baklava-testnet/running-a-validator) section for details on ECDSA and BLS key generation, authorization, and proof of possession.
-
-# Key Security
-
-It is evident that the Account key is the most sensitive key. As it should be used quite infrequently, it is highly recommended for important Account keys (i.e. Validators or accounts with high balances) to remain as secure as possible. At the minimum, we recommend them to be offline, ideally in cold storage or on a hardware wallet.
-
-<!-- NOTE: Unclear if this is the right place for the following table, or if it should exist based on how difficult it will be to accuratly maintain -->
-
-# Key Usage in Protocol
```
Comment (nambrot) [inline packages/docs/node-operations/key-management.md:60]: Did you mean to not copy these sections over?

Comment (nategraf) [reply packages/docs/node-operations/key-management.md:60]: Yes, I decided that they were too low level to be there and I couldn't really think of a good place to put them. These kinds of tables are nice, but bit rot quickly.

**Review by nambrot — APPROVED** [review]

File: packages/docs/getting-started/glossary.md (hunk 1 above)
Comment (yorhodes) [reply packages/docs/getting-started/glossary.md:144]: +1

File: packages/docs/operations-manual/node-upgrades.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+## Upgrading a non-validating node
+
+Use these instructions to update non-validating nodes, such as your account node or your attestaion node on the Baklava testnet. Also use these instructions to upgrade your proxy node, but remember not to stop the proxy of a running validator.
+
+### Pull the latest Docker image
+
+```bash
+export CELO_IMAGE=us.gcr.io/celo-testnet/celo-node:baklava
+export export NETWORK_ID=200110 # Baklava testnet phases 2 and 3
```
Comment (yorhodes) [inline packages/docs/operations-manual/node-upgrades.md:19]: are these documented somewhere?

**Review by yorhodes — APPROVED** [review]

**Pull request merged**